pom.xml

```diff
@@ -311,7 +312,7 @@
           <version>${fabric8-maven-plugin.version}</version>
         </plugin>
         <plugin>
-          <groupId>com.github.kongchen</groupId>
+          <groupId>dk.nykredit.swagger</groupId>
```

Comment:
Previous plugin had problems with generating swagger document. New plugin actually works by simply delegating to Swagger's own libraries.